src/Amazon.Common.DotNetCli.Tools/AssumeRoleMfaTokenCodeCallback.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Amazon.Runtime;
+
+namespace Amazon.Common.DotNetCli.Tools
+{
+    /// <summary>
+    /// Handle the callback from AWSCredentials when an MFA token code is required.
+    /// </summary>
+    internal class AssumeRoleMfaTokenCodeCallback 
+    {
+        AssumeRoleAWSCredentialsOptions Options { get; set; }
```

Comment:
Could be private.  Doesn't need to be/shouldn't be referenced externally.

Reply:
Updated. Keep in mind though the default is private when a visibility modifier is not specified

Reply:
Are you sure?  I thought it was internal.
I prefer it to be explicit anyway.